language/Portuguese_Brazil/strings.po

```diff
@@ -416,30 +416,30 @@ msgstr ""
 
 #: src/frontend/mame/ui/info.cpp:176
 msgid "Completely unemulated features: "
-msgstr ""
+msgstr "Características completamente não emuladas"
```

Comment:
This introduces a list, hence the colon/space after it.  If you don't have space/appropriate punctuation you'll end up with:
```
Características completamente não emuladasREDE
```
Note lack of space.